lib/nimbase.h

```diff
@@ -478,7 +478,9 @@ typedef char* NCSTRING;
   } name = {{length, (NI) ((NU)length | NIM_STRLIT_FLAG)}, str}
 
 /* declared size of a sequence/variable length array: */
-#if defined(__GNUC__) || defined(__clang__) || defined(_MSC_VER)
+#if defined(__cplusplus)
+#  define SEQ_DECL_SIZE 1
```

Comment:
But it is more important to be precise for the backend than it is to adhere to ever-evolving standards. So the correct check might be something like

```
#if defined(__cpluscplus) && defined(__clang__)
  # only clang is so picky about
```